src/js/services/profileService.js

```diff
@@ -165,7 +165,7 @@ angular.module('copayApp.services')
       }
 
       var walletClient = bwcService.getClient();
-      walletClient.createWallet('Personal Wallet', 'me', 1, 1, {
+      walletClient.createWallet(gettext('Personal Wallet'), gettext('me'), 1, 1, {
```

Comment:
No problem. I send you a PR with these changes. Ignore it if you already fixed it.